src/app/ApplicationTemplate.UWP/Views/Content/Settings/SettingsPage.xaml

```diff
@@ -80,7 +80,27 @@
 									 x:Uid="Settings_PrivacyPolicy"
 									 HorizontalAlignment="Left"
 									 Command="{Binding NavigateToPrivacyPolicyPage}" />
-
+					<Grid Margin="0,4"
+						  Background="{StaticResource MaterialSurfaceBrush}">
+						<Grid.ColumnDefinitions>
+							<ColumnDefinition Width="*" />
+							<ColumnDefinition Width="Auto" />
+						</Grid.ColumnDefinitions>
+
+						<TextBlock Text="Dark mode"
+								   Foreground="{StaticResource MaterialOnSurfaceBrush}"
+								   VerticalAlignment="Center" />
+
+						<ToggleSwitch HorizontalAlignment="Right"
```

Comment:
There's a MaterialToggleSwitch style no? Why do we need to specify colors on this instance?